drv/gimlet-hf-server/src/main.rs

```diff
@@ -420,6 +423,54 @@ impl idl::InOrderHostFlashImpl for ServerImpl {
             }
         }
     }
+
+    cfg_if::cfg_if! {
```

Comment:
I see the macro form is needed because of the IDL generation. It would be cool if instead we could configure the generation to not include the hash functionality and use `#[cfg(feature = "hash")]` to eliminate the runtime error. 

Nothing to do now, just some ideas for the future.